sparsecoding/inference.py

```diff
@@ -384,12 +387,18 @@ def infer(self, data, dictionary):
             for t in range(0,self.n_iter):
                 
                 # compute LSM loss for the current iteration
-                loss = self.lsm_Loss(data=data, dictionary=dictionary, coefficients=coefficients, lambdas=lambdas, sigma=self.sigma) 
+                loss = self.lsm_Loss(data=data, 
+                                     dictionary=dictionary, 
+                                     coefficients=coefficients, 
+                                     lambdas=lambdas, 
+                                     sigma=self.sigma
+                ) 
+                print(loss.device)
```

Comment:
This print should probably be commented out

Reply:
oops thanks